website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -77,3 +77,17 @@
         <p data-bind="html: message, attr.class: messageClass"></p>
     </div>
 </div>
+
+ <script>
+    // Make s3 settings variables globally accessible for relevant JS
+    <%
+        import json
+        from website.addons.s3.settings import DEFAULT_BUCKET_LOCATION
+    %>
+    window.contextVars = $.extend(true, {}, window.contextVars, {
+        s3Settings: {
+            defaultBucketLocationValue: ${json.dumps(DEFAULT_BUCKET_LOCATION['value'])},
```

Comment:
After preliminary investigation, it seems that addons settings pages are called via their own inner render() call, and don't have easy access to the parent scope / get_globals dictionary. (see  https://github.com/harryrybacki/osf.io/blob/feature/add_location_based_S3_bucket_creation/website/templates/project/settings.mako#L324 )

One potential workaround (under limited testing) would be to add the following to the settings template: (at https://github.com/HarryRybacki/osf.io/blob/feature/add_location_based_S3_bucket_creation/website/addons/s3/templates/s3_node_settings.mako#L84)

`<% from website.util.sanitize import safe_json as sjson %>`